cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionUtilsTest.java

```diff
@@ -127,6 +129,49 @@ public void testProcessAlterPartitionReassignmentResult() throws Exception {
     EasyMock.reset(result);
   }
 
+  @Test
+  public void testGetInterBrokerReplicaTasksToReexecute() {
+    Set<TopicPartition> partitionsInMovement = Set.of(P0, P1);
+
+    ExecutionProposal proposalForPartition0 = EasyMock.mock(ExecutionProposal.class);
+    ExecutionProposal proposalForPartition1 = EasyMock.mock(ExecutionProposal.class);
+    ExecutionProposal proposalForPartition2 = EasyMock.mock(ExecutionProposal.class);
```

Comment:
Why do we need to mock this -- i.e. Can we create an actual `new ExecutionProposal` for each proposal rather than mocking it?

Reply:
To create an ExecutionProposal object,  there seems to be some other irrelevant/unnecessary parameters I need to provide, e.g. oldLeader, oldReplicas, newReplicas. Also I'm not sure what's a good value for those parameters.